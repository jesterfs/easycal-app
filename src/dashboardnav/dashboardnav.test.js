import React from 'react'
import ReactDOM from 'react-dom'
import { BrowserRouter } from 'react-router-dom'
import DashboardNav from './dashboardnav'

import renderer from 'react-test-renderer';

describe('DashboardNav component', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div')
    ReactDOM.render(
      <BrowserRouter>
        <DashboardNav />
      </BrowserRouter>,
      div
    )
    ReactDOM.unmountComponentAtNode(div)
  })
  
  it('renders the UI as expected', () => {
    const div = document.createElement('div')
    const tree = renderer
      .create(<BrowserRouter>
        <DashboardNav />
      </BrowserRouter>,
      div)
      .toJSON();
    expect(tree).toMatchSnapshot();  
    });

});