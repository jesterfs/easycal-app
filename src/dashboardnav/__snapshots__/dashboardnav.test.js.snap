// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DashboardNav component renders the UI as expected 1`] = `
<div
  className="DashboardNav, group"
>
  <div
    className="item addMemberBtn navbtndiv"
  >
    <a
      href="/addmember"
      onClick={[Function]}
    >
      <button
        className="navbtn"
      >
        Add Team Member
      </button>
    </a>
  </div>
  <div
    className="item"
  >
    <h1>
      EasyCal
    </h1>
  </div>
  <div
    className="item navbtndiv"
  >
    <a
      href="/account"
      onClick={[Function]}
    >
      <button
        className="navbtn"
      >
        Account
      </button>
    </a>
  </div>
</div>
`;
